api/peripheral_api/routes/Speaker.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { speakerQueued } = require('../util/Speaker.js');
+const {
+  verifyToken,
+  checkIfTokenSent,
+} = require('../../util/token-verification');
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
+} = require('../../util/constants').STATUS_CODES;
+const logger = require('../../util/logger');
+
+router.post('/stream', async (req, res) => {
+  /*
+     * How these work with Quasar:
+     * https://github.com/SCE-Development/Quasar/wiki/How-do-Health-Checks-Work%3F
+     */
+  await axios
+    .post(`http://host.docker.internal:18000/stream/?url=${encodeURIComponent(req.query.url)}`)
+    .then(() => {
+      return res.sendStatus(OK);
+    })
+    .catch((err) => {
+      logger.error('Printer SSH tunnel is down: ', err);
+      return res.sendStatus(500);
```

Comment:
add a status code for 500 like `INTERNAL_SERVER_ERROR` in util/constants so we can do

```suggestion
      return res.sendStatus(INTERNAL_SERVER_ERROR);
```